libmultilabel/data_utils.py

```diff
@@ -184,17 +192,14 @@ def load_or_build_label(datasets, label_file=None, silent=False):
     return classes
 
 
-def get_embedding_weights_from_file(word_dict, embed_file, silent=False, normalize=False):
+def get_embedding_weights_from_file(word_dict, embed_file, silent=False): # , normalize=False):
```

Comment:
```suggestion
def get_embedding_weights_from_file(word_dict, embed_file, silent=False):
```
I guess not need to mention normalize here..